scripts/test-infra/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+make build -e
```

Comment:
If the variable initializations in Makefile is changed to ?= we could simply use make image without -e

Reply:
What about this?